.taskfiles/test.yml

```diff
@@ -86,8 +87,8 @@ tasks:
     vars:
       count: '{{ .count | default "10" }}'
     cmds:
-      - gotestsum --hide-summary=skipped --junitfile=unit-tests.xml --raw-command cat coverage/{{.product}}-all.json
-      - echo "Slowest {{.count}} tests:" && cat coverage/{{.product}}-all.json | gotestsum tool slowest | head -n {{.count}} | sed -e 's|{{.package}}/||g'
+      - gotestsum --hide-summary=skipped --junitfile=unit-tests.xml --raw-command cat coverage/*.json
+      - echo "Slowest {{.count}} tests:" && cat coverage/*.json | gotestsum tool slowest | head -n {{.count}} | sed -e 's|{{.package}}/||g'
 
```

Comment:
The above change makes `test:integration` and `test:report` (slowest tests) work together.

`cat coverage/*.json | gotestsum tool slowest`

By default gotestsum reports tests slower than 100ms, reports ~150 tests out of the set of ~2600.